provisioning/service/samples/GroupCertificateVerificationSample/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.Azure.Devices.Provisioning.Samples
+{
+    public class Program
+    {
+        private const string VerificationCertificatePath = "verificationCertificate.cer";
```

Comment:
Please add <> in the verification cert path to indicate that users have to replace with their own values.  Our other samples have something like:
private const string DeviceConnectionString = "<replace>";